pkg/db/jsonType_test.go

```diff
@@ -0,0 +1,98 @@
+package db_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/justtrackio/gosoline/pkg/db"
+	"github.com/stretchr/testify/require"
+)
+
+func TestJSONType(t *testing.T) {
+	t.Parallel()
+
+	v := struct{}{}
```

Comment:
I would maybe store something in the struct instead of using an empty struct. For example, doing nothing in `Scan` would be a correct implementation for this test